pymc/distributions/timeseries.py

```diff
@@ -143,17 +148,19 @@ class GaussianRandomWalk(distribution.Continuous):
 
     rv_op = gaussianrandomwalk
 
-    def __new__(cls, name, mu=0.0, sigma=1.0, init=None, steps: int = 1, **kwargs):
+    def __new__(cls, name, mu=0.0, sigma=1.0, init=None, steps=None, **kwargs):
```

Comment:
I think I asked this question before but do `__new__` and `dist` have to have almost the same signature (except for size)?

Reply:
They can have the same signature. I don't know if other distributions usually show or hide size in kwargs. Only thing we care about here is getting `init`

Reply:
Is it ever the case theyll have vastly different signatures?

Reply:
`__new__` is only used internally. Usually we just need to take some arguments for checks like the one here. Otherwise we leave it unchanged/undefined.